corehq/apps/api/resources/v0_5.py

```diff
@@ -986,3 +991,43 @@ def prepend_urls(self):
             url(r"^(?P<resource_name>%s)/(?P<app_id>[\w\d_.-]+)/(?P<xmlns>[\w\d_.-]+)" % self._meta.resource_name,
                 self.wrap_view('dispatch_list'))
         ]
+
+
+class ODataCaseFromExportInstanceResource(v0_4.CommCareCaseResource):
+
+    config_id = None
+
+    def dispatch(self, request_type, request, **kwargs):
+        if not toggles.ODATA.enabled_for_request(request):
+            raise ImmediateHttpResponse(response=HttpResponseNotFound('Feature flag not enabled.'))
```

Comment:
You could probably stick this in `ODataAuthentication`

Reply:
This feature flag will be dropped soon, so I'm not worrying too much about where to put it.